### ISSUE TITLE
Crashes in Vite

### ISSUE THREAD

**Issue by patarapolw** [eta-dev/eta#122]
**Describe the bug**

Importing in Vite for this very simple code crashes.

```ts
import Eta from 'eta'

Object.assign(window, { Eta, })
```

```
'resolve' is not exported by __vite-browser-external, imported by node_modules/eta/dist/eta.es.js
file: /home/polv/projects/rep2recall-min/packages/frontend/node_modules/eta/dist/eta.es.js:2:9
1: import { existsSync, readFileSync } from 'fs';
2: import { resolve, dirname, extname } from 'path';
            ^
3: 
4: /*! *****************************************************************************
error during build:
Error: 'resolve' is not exported by __vite-browser-external, imported by node_modules/eta/dist/eta.es.js
    at error (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:7917:30)
    at Module.error (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:9860:16)
    at Module.traceVariable (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:10241:29)
    at ModuleScope.findVariable (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:9018:39)
    at FunctionScope.findVariable (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:3240:38)
    at ChildScope.findVariable (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:3240:38)
    at Identifier.bind (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:4169:40)
    at CallExpression.bind (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:2490:23)
    at CallExpression.bind (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:6589:15)
    at BinaryExpression.bind (/home/polv/projects/rep2recall-min/packages/frontend/node_modules/rollup/dist/shared/rollup.js:2490:23)
error Command failed with exit code 1.
```

I doubt this also crashes in Webpack, Rollup and ESBuild.

It doesn't seem to crash in Snowpack, for some reasons.

[Documentation](https://eta.js.org/docs/learn/install) is also lacking on this.

I avoid CDN, because I want to make an offline Electron-like app.

**Expected behavior**

No crashes.

**Package & Environment Details**
 - node v14.17.0
 - yarn 1.22.10
 - vite 2.3.7
 - eta 1.12.2

**Additional context**

I believe I have to tweak something in Webpack, just like ejs. (But I am not using Webpack right now.)

**Comment by shadowtime2000** [comment]
I edited the rollup build config a while back to have a browser ES version shipped, but I believe I reverted it to make sure I didn't have any new stuff in the latest patch release. I think at this point at time, once Eta makes fully compatible with the latest Deno version I can try editing the build config to this.

**Comment by plashenkov** [comment]
The same problem. Is there a workaround?

**Comment by shadowtime2000** [comment]
Would be resolved by #154

**Comment by shadowtime2000** [comment]
@plashenkov @patarapolw This will be resolved in the next update, which will be a patch release. You should be able to import using something like `eta/dist/browser/eta.es.dev.js` or `eta/dist/browser/eta.es.min.js`.
